tests/unit/consensus/test_helpers.py

```
@@ -582,27 +657,31 @@ async def fake_snapshot():
                     "data": {
                         "state": {"accepted": {}, "finalized": {}},
                         "code": "contract_code",
                     },
+                    "balance": 0,
                 }
             }
         )
+    if accounts_manager is None:
+        accounts_manager = AccountsManagerMock(current_state_db)
+
+    chain_snapshot_factory = lambda session: chain_snapshot
```

Ruff (0.11.9)

Do not assign a `lambda` expression, use a `def`

Rewrite `chain_snapshot_factory` as a `def`

(E731)